app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,26 @@ def show
     @user = User.find(params[:user_id])
     @post = @user.posts.find(params[:id])
   end
+
+  def new
+    @user = User.find(params[:user_id])
+    @add_new_post = @user.posts.build
+  end
+
+  def create
+    current_user = User.find(params[:user_id])
```

Comment:
- [ ] @AlselwiAisha Kindly consider adding the current_user method in your application controller and then use it here, instead of creating a variable as per the project instructions
![image](https://github.com/AlselwiAisha/BlogApp/assets/61408860/c899303b-686b-4c06-a3de-54ff168a8a97)